shell/platform/windows/flutter_windows_engine_unittests.cc

```diff
@@ -645,6 +645,7 @@ TEST_F(FlutterWindowsEngineTest, TestExit) {
   MockFlutterWindowsView view(std::move(window_binding_handler));
   view.SetEngine(builder.Build());
   FlutterWindowsEngine* engine = view.GetEngine();
+  engine->OnServiceBindingsRegistered();
```

Comment:
Can you maybe add a test that makes sure it doesn't handle `WM_CLOSE` before `OnServiceBindingsRegistered` is called?